mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/LocalizedStrings.kt

```diff
@@ -77,6 +77,10 @@ interface LocalizedStrings {
   val gamePromoteTitle: String
   val gamePromoteConfirm: String
 
+  val gameSnackBarIllegalAction: String
+  val gameSnackBarInternalFailure: String
+  val gameSnackBarUnknownCommand: String
```

Comment:
For a next PR, it could be good to have additional information about the specific error (by taking a parameter)

Reply:
Hum, something like this : `a2-a3 is an invalid action` ?

Reply:
Yes, because the voice recognizer could have understood another valid command that is not a valid action. (And the player could not know)

Reply:
Added in the issue https://github.com/epfl-SDP/android/issues/341#issuecomment-1125244852